src/md/contributing.md

```diff
@@ -153,6 +153,10 @@ Thanks for contributing!
   - Ensure tests are added/modified for new features or fixes
   - Ensure appropriate JavaDoc comments have been added
   - Verify contributions don't depend on incompatible licences (see https://www.apache.org/legal/resolved.html#category-x)
+1. If the CI build fails because of the presence of a CVE vulnerability, further analysis needs to be performed
+  - If the CVE is unrelated to the changes in the PR i.e., the changes in the PR are not the cause then it can be merged
+  - If the vulnerability is in a dependency added by the PR then the committer should ask the contributor to address it. If there are no good alternatives, then a discussion should happen in the security list whether to allow the PR before it can be merged
```

Comment:
Please add instructions for whitelisting the dependency with CVE in case exception is granted.